data/sql/updates/pending_db_world/rev_1640946362151198685.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1640946362151198685');
+
+DELETE FROM `creature_loot_template` WHERE (`Entry` = 13036) AND (`Item` BETWEEN 18255 AND 18533);
```

Comment:
Please adjust per my information based on sniffs and udb comparisons from the sniff team collective.

https://github.com/azerothcore/azerothcore-wotlk/pull/9947#issuecomment-1003409087